gh-107583: Update Doc/howto/unicode.rst (#109501)
python/cpython: Doc/howto/unicode.rst

any encoding if you declare the encoding being used.  This is done by including
a special comment as either the first or second line of the source file::

    # coding: latin-1

    u = 'abcdé'
    print(ord(u[-1]))

Python looks for patterns such like ``coding: latin-1``, ``coding=latin-1`` or

> ``-*- coding: latin-1 -*-`` in the comment. See :ref:`encodings` for detailed

Reviewer: ```suggestion
``-*- coding: latin-1 -*-`` in the comment. See :ref:`encodings` for a detailed
```
Grammar nit

explanation.

If you don't include such a comment, the default encoding used will be UTF-8 as

> already mentioned.

Reviewer 2: Please keep the reference to PEP 263, since this explains how it all works.

<https://downloads.egenix.com/python/Unicode-EPC2002-Talk.pdf>`_ at
EuroPython 2002.  The slides are an excellent overview of the design of Python
2's Unicode features (where the Unicode string type is called ``unicode`` and

> literals start with ``u``). (Notice: Python 2 is no longer supported, just keep it

Reviewer 2: Better: (Note: Python 2 is no longer supported. We just keep this reference for historic reasons.)

> literals start with ``u``). (Notice: Python 2 is no longer supported, just keep it
> here for historical interest.)

Reviewer: ```suggestion
literals start with ``u``). (Note: Python 2 is no longer supported.
This reference is preserved for historic reasons.)
```
A slightly different suggestion to the above.
